app/weeklyPicks/page.tsx

```diff
@@ -84,22 +82,30 @@ export default function WeeklyPickForm() {
         window.localStorage.setItem('team', data.type);
       }
 
-      const user = await account.get();
-      const response = await getUserWeeklyPick({
-        userId: user.$id,
-        weekNumber: '6622c75658b8df4c4612',
-      });
-
       const findTeamID = NFLTeams.find(
         (ele) => ele.teamName.toLowerCase() === teamSelect,
       );
 
-      response[user.$id] = { team: findTeamID?.$id, correct: true };
+      let appendNewResult;
+      if (
```

Comment:
This logic should not be here. We are allowing the user to still make changes to their pick. There's a feature where we will lock the user from making changes, but that was not in scope for the MVP.